docs/StardustDocs/topics/Usage-with-Kotlin-Notebook-Plugin.md

```diff
@@ -0,0 +1,47 @@
+# Usage with Kotlin Notebook Plugin
+
+The [Kotlin Notebook Plugin for IntelliJ IDEA](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook), combined with Kotlin DataFrame, offers powerful data analysis capabilities within an interactive environment. Here are key features:
+
+### Visual Data Exploration
+**Page through your data**:
+The pagination feature lets you move through your data one page at a time, making it possible to view large datasets.
+
+**Sort by column with a single click**:
+You can sort any column with a click. This is a convenient alternative to using sortBy in separate cells.
+
+**Go straight to the data you need**:
+You can jump directly to a particular row or column if you want something specific.
+This makes working with large datasets more straightforward.
+<video src="https://github.com/user-attachments/assets/aeae1c79-9755-4558-bac4-420bf1331f39" controls></video>
+
+### Drill down into nested data
+When your data has multiple layers, like a table within a table, you can now click to view these details directly.
```

Comment:
click on what? :) I'd specify "click on a cell containing a nested table" or something like that